### ISSUE TITLE
`mfind ... DIR1 NOSUCHDIR DIR2` errors with "NotFoundError" but doesn't specify which

### ISSUE THREAD

**Issue by trentm** [joyent/node-manta#224]
```
$ mfind -t o /trent.mick/public/tmp /trent.mick/public/nada /trent.mick/stor/tmp
mfind: NotFoundError
```

would be very helpful if the error included the dir that wasn't found.

**Comment by davepacheco** [comment]
I took a quick look at this.  Theoretically this would be easy, since VError is designed exactly to be able to wrap lower-level with an error providing more detail.  I ran into a few problems:

- VError expects callers to just print its message, but node-manta prints out err.toString().  So you get an error that says "VError: " in it, which is confusing.
- I tried changing the surrounding code to print the error's message instead, but it looks like some of the node-manta errors don't have messages because I just got: `mfind: "/asdf": ` (i.e., no explanation of the actual error at all).  That also suggests that if this _weren't_ a VError (because it was some other code path), we'd have gotten no error message at all.

Besides that, the error printing code is command-specific, at least for mfind.  Overall, It looks like some cleanup will be needed to make this work.

**Comment by davepacheco** [comment]
The errors-having-no-messages issue is #198, and I do believe this Error is constructed by node-restify.  The server is returning an error with code ResourceNotFound, but the error reported is a NotFoundError.  I can't find any real references to that anywhere in the client (including node modules), so I think it's constructed from the HTTP status line, which is "404 Not Found".

**Comment by arekinath** [comment]
Released in 4.2.0
